docs/deployment/helm.md

```diff
@@ -132,7 +132,7 @@ API's you need to install the prometheus operator in your cluster.
 | `master.service.type`      | string  | ClusterIP                               | NFD master service type. **NOTE**: this parameter is related to the deprecated gRPC API and will be removed with it in a future release |
 | `master.service.port`      | integer | 8080                                    | NFD master service port. **NOTE**: this parameter is related to the deprecated gRPC API and will be removed with it in a future release |
 | `master.resources`         | dict    | {}                                      | NFD master pod [resources management](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) |
-| `master.nodeSelector`      | dict    | {}                                      | NFD master pod [node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) |
+| `master.nodeSelector`      | dict    | {limits: {cpu: 300m, memory: 1Gi}, requests: {cpu: 100m, memory: 128Mi}} | NFD master pod [node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) |
```

Comment:
perhaps this should be `master.resources` ?

Reply:
nit: maybe it is just a personal taste, but it is seems it is easier to read if you could have two columns for `x.resources.limits` and `x.resources.requests` instead of merging them into a single column.

Reply:
> perhaps this should be `master.resources` ?

akhh forgot to check this, yes it should be resources

Reply:
> nit: maybe it is just a personal taste, but it is seems it is easier to read if you could have two columns for `x.resources.limits` and `x.resources.requests` instead of merging them into a single column.

Yes I think it should be better, thanks.